--- fontconfig-2.11.95/src/fcint.h
+++ fontconfig-2.11.95/src/fcint.h
@@ -89,7 +89,7 @@
 #define FC_DBG_LANGSET	2048
 #define FC_DBG_MATCH2	4096

-#define _FC_ASSERT_STATIC1(_line, _cond) typedef int _static_assert_on_line_##_line##_failed[(_cond)?1:-1] FC_UNUSED
+#define _FC_ASSERT_STATIC1(_line, _cond)
 #define _FC_ASSERT_STATIC0(_line, _cond) _FC_ASSERT_STATIC1 (_line, (_cond))
 #define FC_ASSERT_STATIC(_cond) _FC_ASSERT_STATIC0 (__LINE__, (_cond))
